python/GangaLHCb/test/GPI/TestExternalGaudiExec.py

```diff
@@ -110,6 +110,10 @@ def testSubmitJobComplete(self):
         from Ganga.GPI import jobs
 
         j = jobs[-1]
+        j = j.copy()
+        j.unprepare()
+        j.prepare()
+        j.submit()
```

Comment:
There seems to be a problem in that the SharedPath object, when read back from disk, is assigned a new directory name. So if I prepare a job, quit Ganga and then start again, the application object points to an empty directory and the job fails. So this is a potentially serious problem (that we need to understand), but not the propose of this test.